docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,99 @@
+
+`viam-server` connects your machine to the Viam app and to SDK clients.
+
+See [Viam architecture](/operate/reference/architecture/) for details on how Viam works.
+
+## What is a machine?
+
+A _machine_ is a computer (often a single-board computer like a Raspberry Pi or Jetson) or microcontroller and all the hardware attached to it, as well as the software running on it.
+You can think of one machine as representing one device, or one robot.
+Each machine runs an instance of `viam-server`.
+
+When you create a new machine in the Viam app, Viam generates a unique set of credentials for that machine that connect the physical machine to its instance in the Viam app.
+
+## What platforms does Viam run on?
+
+Viam can run on any computer that runs one of the following operating systems:
+
+- Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
+- macOS
+- Windows
+- 32-bit microcontrollers (limited support)
+
+Examples of computers that can run Viam:
+
+{{< board-carousel >}}
+
+## What hardware does Viam support?
+
+In short, anything.
+
+Viam supports a wide variety of sensors, cameras, and other physical hardware, with APIs for each of the following types of hardware:
+
+{{< cards >}}
+{{% relatedcard link="/dev/reference/apis/components/arm/" %}}
+{{% relatedcard link="/dev/reference/apis/components/base/" %}}
+{{% relatedcard link="/dev/reference/apis/components/board/" %}}
+{{% relatedcard link="/dev/reference/apis/components/button/" %}}
+{{% relatedcard link="/dev/reference/apis/components/camera/" %}}
+{{% relatedcard link="/dev/reference/apis/components/encoder/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gantry/" %}}
+{{% relatedcard link="/dev/reference/apis/components/gripper/" %}}
+{{% relatedcard link="/dev/reference/apis/components/input-controller/" %}}
+{{% relatedcard link="/dev/reference/apis/components/motor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/movement-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/power-sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/sensor/" %}}
+{{% relatedcard link="/dev/reference/apis/components/servo/" %}}
+{{% relatedcard link="/dev/reference/apis/components/switch/" %}}
+{{% relatedcard link="/dev/reference/apis/components/generic/" %}}
+{{< /cards >}}
+
+These standardized APIs are implemented by {{< glossary_tooltip term_id="module" text="modules" >}} that provide drivers for specific models of hardware.
+
+Any hardware that is not already supported by a Viam module can be added into Viam's system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
+
+Viam also supports various software services such as [data capture](/data-ai/capture-data/capture-sync/) and [computer vision](/dev/reference/apis/services/vision/), designed to integrate seamlessly with the hardware driver modules.
+
+## How do I get started?
```

Comment:
As a general comment, is there a way to make the links in this list have the exact text of things in the left nav?

For example, instead of a link on all of "Set up your machine and connect it to the Viam app", the link could be on just "Set up a machine"

Similarly for the other steps

Reply:
The former sounds good - just moving which words the link is on. I was thinking it's a nice visual clue that the left nav and these next steps are the same

Reply:
Okay sounds like I've already applied this one then cool!

Reply:
Okay I did tweak a couple, but since set up goes to either of two pages it's not a perfect match and neither is integrate other hardware, though that one contains the phrase. LMK if you think it needs to more strictly match.

Reply:
Thanks for changing those. I was just thinking about this - were we going to move the two setup pages under a common left nav title "Set up a machine"? I still think that's a good idea, since other than the two setup pages, the left nav is meant to be progressed through linearly

Reply:
(per offline convo leaving as-is for now but open to changing)